plugin.video.crackle/resources/lib/globals.py

```diff
@@ -0,0 +1,198 @@
+import sys, os
+import xbmc, xbmcplugin, xbmcgui, xbmcaddon
+import urllib, urllib2
+import json
+import base64, hmac, hashlib
+from datetime import datetime
+
+addon_handle = int(sys.argv[1])
+ADDON = xbmcaddon.Addon()
+ROOTDIR = ADDON.getAddonInfo('path')
+FANART = ROOTDIR+"/resources/media/fanart.jpg"
+ICON = os.path.join(ROOTDIR,"/resources/media/icon.png")
+
+
+#Addon Settings
+LOCAL_STRING = ADDON.getLocalizedString
+UA_CRACKLE = 'Crackle/7.60 CFNetwork/808.3 Darwin/16.3.0'
+UA_WEB = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/57.0.2987.110 Safari/537.36'
+UA_ANDROID = 'Android 4.1.1; E270BSA; Crackle 4.4.5.0'
+PRIVATE_KEY = 'MIRNPSEZYDAQASLX'
```

Comment:
I don't know how sensitive this private key is but I'd recommend to encode it at least with `base64` encoding to prevent "naked eye" reading and/ or webbot parsing from sources.